### PR TITLE
D-Link.TFTPServer version 1.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108293)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @upintheairsheep,

Does this package have a website or other link? I'm having trouble finding this package on D-Link's website.

**Comment by upintheairsheep** [comment]
> Hi @upintheairsheep,
> 
> Does this package have a website or other link? I'm having trouble finding this package on D-Link's website.

Taken from their official Downloads directory https://ftp.dlink.ru/pub

**Review by ItzLevvie — APPROVED** [review]

File: manifests/d/D-Link/TFTPServer/1.0/D-Link.TFTPServer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: D-Link.TFTPServer
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: D-Link
+PackageName: TFTP Server
```
Comment (stephengillie) [inline manifests/d/D-Link/TFTPServer/1.0/D-Link.TFTPServer.locale.en-US.yaml:8]:
```suggestion
PackageName: TFTP Server
PackageUrl: https://ftp.dlink.ru/pub/
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**